beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/hive/HiveClient.java

```diff
@@ -117,6 +117,9 @@ public boolean tableExists(String databaseName, String tableName) {
         return new HashMap<>();
       }
       return table.getParameters();
+    } catch (NoSuchObjectException e) {
+      log.warn("The table {}.{} does not exist", databaseName, tableName);
```

Comment:
Maybe a bit more informative this log? Could we say that we are going to remove table properties?

Reply:
this is a general hive util method, so it may not apply to other calls